internal/app/organisation_store.go

```diff
@@ -167,6 +174,17 @@ func (s *organisationStore) AllLPAs(ctx context.Context) ([]actor.DonorProvidedD
 	return donors, nil
 }
 
+func (s *organisationStore) SoftDelete(ctx context.Context) error {
+	organisation, err := s.Get(ctx)
```

Comment:
you have the organisation when SoftDelete is used, so could pass it in instead of re-getting

Reply:
Good point 👍